src/pages/Faq/Faq.tsx

```diff
@@ -1,5 +1,79 @@
 import './Faq.scss';
+import { FaqPageSlice } from './components/FaqPageSlice';
 
 export const Faq = () => {
-  return <></>;
+  return (
+    <>
+      <div className="faq-page">
+        <div className="right">
+          <div className="general-information">
+            <div className="right-header">
+              <h1>Generel Information</h1>
+            </div>
+            <div className="text-box-right">
+              <div className="text-box">
+                <FaqPageSlice
```

Comment:
could you create the "faqs" array and then do something like this?
`{
  faqs.map(faq => <div className="text-box">
         <FaqItem header={faq.header} text={faq.text}>
    </div>
  )
}`